src/transformStateWithClones.js

```diff
@@ -66,6 +66,35 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
-}
+  let stateClone = Object.assign({}, state);
+  const arrayOfTransforms = [];
+  const newOperations = {
+    add: 'addProperties',
+    remove: 'removeProperties',
+    clear: 'clear',
+  };
+
+  for (const keys in transforms) {
+    switch (transforms[keys].operation) {
+      case newOperations.add:
+        stateClone = Object.assign(stateClone, transforms[keys].properties);
```

Comment:
```suggestion
         Object.assign(stateClone, transforms[keys].properties);
```
Object.assign присвоює все в той об'єкт який був переданий першим параметром. Не треба додаткового присвоєння